libs/community/langchain_community/chat_models/anyscale.py

```diff
@@ -152,6 +146,8 @@ def validate_environment_override(cls, values: dict) -> dict:
                 }
                 values["client"] = openai.OpenAI(**client_params).chat.completions
             else:
+                values["openai_api_base"] = values["anyscale_api_base"]
+                values["openai_api_key"] = values["anyscale_api_key"].get_secret_value()
```

Comment:
this is changing to a SecretStr as well soon too

Reply:
Sorry, I don't understand this comment. In `openai.py`, the `values["openai_api_key"]` is assigned as string as well